backends/tfhe-cuda-backend/cuda/src/fft128/fft128.cuh

```diff
@@ -356,6 +375,70 @@ batch_NSMFFT_128(double *in_re_hi, double *in_re_lo, double *in_im_hi,
   }
 }
 
+template <class params, sharedMemDegree SMD>
+__global__ void batch_NSMFFT_strided_128(double *d_in, double *d_out,
```

Comment:
Why is there strided in the name?

Reply:
depends on how what pattern was used to store `re_hi, re_lo, im_hi and im_lo` parts we either call
`batch_NSMFFT_strided_128` or `batch_NSMFFT_128`